### PR TITLE
Add more to the Tooling section

### PR DESCRIPTION
**Pull request by adpi2**
Move the following pages to the Tooling section:
- Tour of the Migration Tools
- Scala 3 Migration Mode
- Scala 3 Syntax Rewriting

Scala3-migrate is renamed Scala 3 Migrate Plugin

The Tour of the Migration Tools is reorganized and updated

### CODE REVIEW

File: docs/compatibility.md [hunk 1]
```diff
@@ -20,9 +20,9 @@ There is an exception though, which is the new metaprogramming framework that re
 Further explanations are given down below.
 
 Setting metaprogramming aside, a Scala 2.13 source code can rather easily be translated into Scala 3.
```
[the remark this answers is not in the document]
Comment (mlachkar) [reply docs/compatibility.md:22]: this suggestion is on multiple lines.

File: docs/compatibility.md [hunk 2]
```diff
@@ -20,9 +20,9 @@ There is an exception though, which is the new metaprogramming framework that re
...
-Once you have it, you will be able to use the new powerful features of Scala 3, which have no equivalent in Scala 2.
-Obviously, by doing so the sources won't compiled in Scala 2.13 anymore.
-But amazingly, once compiled by Scala 3, this new module can be consumed as a dependency by the Scala 2.13 compiler.
+Once you have it translated, you will be able to use the new powerful features of Scala 3, which have no equivalent in Scala 2.
+By doing so the sources won't compiled in Scala 2.13 anymore.
```
Comment (mlachkar) [inline docs/compatibility.md:24]: compiled => compile

**Comment by mlachkar** [comment]
it's just propositions

File: docs/compatibility.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (adpi2) [reply docs/compatibility.md:22]: thanks

File: docs/compatibility.md [hunk 3]
```diff
@@ -20,9 +20,9 @@ There is an exception though, which is the new metaprogramming framework that re
 Further explanations are given down below.
 
 Setting metaprogramming aside, a Scala 2.13 source code can rather easily be translated into Scala 3.
-Once you have it, you will be able to use the new powerful features of Scala 3, which have no equivalent in Scala 2.
-Obviously, by doing so the sources won't compiled in Scala 2.13 anymore.
-But amazingly, once compiled by Scala 3, this new module can be consumed as a dependency by the Scala 2.13 compiler.
+Once you have it translated, you will be able to use the new powerful features of Scala 3, which have no equivalent in Scala 2.
```
[the remark this answers is not in the document]
Comment (adpi2) [reply docs/compatibility.md:23]: yes it is better

File: docs/compatibility.md (hunk 2 above)
Comment (adpi2) [reply docs/compatibility.md:24]: thanks

File: docs/compatibility.md [hunk 4]
```diff
@@ -4,8 +4,8 @@ title: Compatibility Reference
 ---
 Scala 3 is a game changer in terms of compatibility in the Scala ecosystem that will greatly improve the day-to-day experience of every Scala programmer, starting from the migration.
 
-In this page you will learn about the compatibility between Scala 2.13 and Scala 3 at different levels of the program lifetime.
-You will then find how to benefit from these levels of compatibility in concrete examples of dependencies between the two versions.
+In this page you will learn about the compatibility between Scala 2.13 and Scala 3 at different stages of the program lifetime.
+You will then find how to benefit from these levels of compatibility in concrete examples of dependencies.
```
[the remark this answers is not in the document]
Comment (adpi2) [reply docs/compatibility.md:8]: thanks

**Review by mlachkar — APPROVED** [review]

**Pull request merged**